frontends/election-manager/src/app_root.tsx

```diff
@@ -79,7 +78,7 @@ export function AppRoot({
 
   const store = useElectionManagerStore({
     logger,
-    storage,
+    backend,
   });
```

Comment:
Consider using a context provider to make the store available throughout the app. Maybe something for a later PR

Reply:
Were you thinking it would be like this?

```ts
// app.tsx
function App() {
  const logger = …;
  const backend = …;

  const store = useElectionManagerStore({ backend, logger });

  return (
    <StoreContext.Provider value={{ store }}>
      …
    </StoreContext.Provider>
  );
}

// my_component.tsx
function MyComponent() {
  const { store } = useContext(StoreContext);
}
```

Or like this?

```ts
// app.tsx
function App() {
  const logger = …;
  const backend = …;

  return (
    <StoreContext.Provider value={{ backend, logger }}>
      …
    </StoreContext.Provider>
  );
}

// use_election_manager_store.ts
function useElectionManagerStore() {
  const { logger, backend } = useContext(StoreContext);

  return …;
}

// my_component.tsx
function MyComponent() {
  const store = useElectionManagerStore();
}
```

Reply:
I was thinking along the lines of the first example, but with a little more sugar:

```ts
// my_component.tsx
function MyComponent() {
  const store = useStore();
}
```

Another option entirely would be to create custom hooks for each query/mutation in the store (which is the pattern we use in Arlo). Just throwing that on the table for consideration, I don't necessarily think it's better.

Reply:
Will think about this more for a follow-up.